packages/plugin-ext/src/main/browser/view/tree-view-decorator-service.ts

```diff
@@ -0,0 +1,35 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
```

Comment:
@xcariba was this file copied from somewhere, else you should put `2021` and you company if it exists.

Reply:
Yeah, thanks, fixed